layouts/community/ergodox/azurefog/keymap.c

```diff
@@ -0,0 +1,385 @@
+#include "ergodox_ez.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```